governance/blogpost-process.md

```diff
@@ -0,0 +1,131 @@
+2. **Consider the target audience**: Identify the intended audience for the blog post. Is it written for beginner developers, experienced programmers, or a specific niche? Evaluate whether the content is appropriate and understandable for the target readership.
+3. **Assess the title and introduction**: Review the title and introduction to determine if they effectively capture the reader's attention and provide a clear overview of what the post is about. A concise and informative title, along with a compelling introduction, can engage readers and set expectations.
+4. **Check the structure and flow**: Evaluate the overall structure and flow of the blog post. Look for a logical organization, clear headings, and subheadings. Ensure that the content follows a coherent progression and is easy to follow.
+5. **Analyze the technical accuracy**: Examine the technical accuracy of the content. Check if the information provided is correct, up-to-date, and follows best practices. Evaluate whether the author demonstrates a deep understanding of the topic and provides accurate explanations and examples.
+6. **Evaluate the clarity and depth**: Assess the clarity of the explanations and instructions provided in the blog post. Determine if the author effectively communicates complex concepts and techniques. Look for in-depth explanations and examples that help readers understand the subject matter.
+7. **Assess code examples and snippets**: Evaluate the code examples and snippets included in the blog post. Check for correctness, readability, and adherence to coding standards. Consider if the code is well-commented and includes any necessary error handling or exception handling.
+8. **Look for visual aids**: Assess if the blog post incorporates relevant visual aids such as diagrams, charts, or screenshots. Evaluate whether these visuals enhance the content and make it easier to understand or follow along with.
+9. **Check for readability and formatting**: Consider the readability and formatting of the blog post. Look for clear and concise sentences, proper use of technical terms, and consistent formatting of code snippets. Ensure that the language used is accessible to the target audience and avoids unnecessary complexity or ambiguity.
+10. **Provide constructive feedback**: Offer specific and constructive feedback to help the author improve the blog post. Identify areas that need clarification, provide suggestions for additional code examples or explanations, and highlight any potential errors or omissions. Be supportive in your critique and focus on helping the author enhance the technical quality and readability of their content.
+
+Follow this [guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for an introduction how to review Pull Requests (PRs) on GitHub.
```

Comment:
@mkorbi That's understandable. This TAG should determine what their preference is, and it may be a combination of the two. There have been concerns in the past where working in hackmd or google docs lose the recognition of the contributor in doing the work once moved to a PR in git, (git has no ability to track the specific contributions unless they occur there by the individual through a commit).  A modification to resolve this may be that the PR and final content should include a link to the original working document to preserve that history and make it available for others. This also helps immensely when planning version updates of papers or other content as you have the source doc to go back and learn decisions or discussions from.

Reply:
As long as 1) the people contributing to the blog are credited in some way and 2) we have a review process, that in the end is in git for tracking purposes, I'm happy with any solution.  It may be that contributors have feelings and may start in a google doc anyway as they flesh out their voice.  Basically, both of you have valid points, but I'm not heavily invested in one over the other.  The not-available in some other countries is a concern though, but maybe doesn't matter if all interested parties are not in those countries.  We could put in process guidelines and leave the specifics up to the writer?

Reply:
> We could put in process guidelines and leave the specifics up to the writer?

Agree, we can be flexible about it.

Reply:
One more note to take into consideration based on my experiences:  merging changes with the landscape doc via github, especially in the 11th hour, was messy for me because I had to keep going back and forth between PRs and resolving conflicts (was incredibly thankful for the contributions, but the reality was messy) and hoping to the stars I didn't miss anything (hence all the review requests).  I'll also note that sometimes people don't get to things until the 11th hour, no matter when that hour is, so maybe that isn't really a consideration.

Reply:
I added that we are flexible in how blog posts are reviewed [3eb7d89](https://github.com/cncf/tag-env-sustainability/pull/105/commits/3eb7d89eadcd543481fd2c61eef26d3ed7102177)

Reply:
> I think this is not a problem. In projects we review code with large PRs via GitHub and that works IMO quite well.
@leonardpahlke At the same time we discuss how to lower the wall for nontechnical contribution

> Contributions are not lost. This has been the case in the past when using Hackmd / Google Docs. Adding contributors in a note to the document is fine, but it is better to use Git from the beginning IMO.

We might add two lines in the document: Authors & Reviewers
Everyone can add their name, no one gets lost. Our biggest concern shouldn't be to lose contributions and their contributors. Prio 1 should be to get the work done in the best collaborative way possible.

Reply:
> We might add two lines in the document: Authors & Reviewers

Already, captured ✅ https://github.com/cncf/tag-env-sustainability/blob/main/.github/ISSUE_TEMPLATE/blog.md